profile/README.md

```diff
@@ -5,6 +5,7 @@ You may learn more about our tools and our work in progress by checking out the
 following links:
 
 - [The epiverse-trace R-universe](https://epiverse-trace.r-universe.dev/)
+- [The epiverse-trace developer corner](https://epiverse-trace.github.io/)
```

Comment:
probably change to "developer space" instead of "corner"? for me "corner" means to be apart from the centre of attention, hehe. the change would be also consistent with the webpage header.